spec/services/search/query_builders/classified_listing_spec.rb

```diff
@@ -16,11 +16,25 @@
 
   describe "#as_hash" do
     it "applies TERM_KEYS from params" do
-      params = { category: "cfp", tags: ["beginner"], contact_via_connect: false }
+      params = { category: "cfp", tags: %w[beginner intermediate professional], contact_via_connect: false }
+      filter = described_class.new(params: params)
+      exepcted_filters = [
+        { "terms" => { "category" => ["cfp"] } },
+        { "terms" => { "tags" => %w[beginner intermediate professional] } },
+        { "terms" => { "contact_via_connect" => [false] } },
+        { "terms" => { "published" => [true] } },
+      ]
+      expect(filter.as_hash.dig("query", "bool", "filter")).to match_array(exepcted_filters)
+    end
+
+    it "applies TERM_KEYS from params with boolean mode" do
+      params = { category: "cfp", tags: %w[beginner intermediate professional], contact_via_connect: false, tag_boolean_mode: "all" }
       filter = described_class.new(params: params)
       exepcted_filters = [
```

Comment:
small typo here `exepcted_filters`

Reply:
It's spelled the same at various parts in the codebase. Thought it was some kind of convention being followed and intentionally spelled like that.

some places i have found apart from this spec file

spec/services/search/query_builders/feed_content_spec.rb
spec/services/search/query_builders/reaction_spec.rb
spec/services/search/query_builders/user_spec.rb

will create a new PR to fix the typos if required.

Reply:
thank you1